solar_car_process.py

```diff
@@ -23,5 +23,5 @@ def restart(self):
         self.timesRestarted += 1
 
     def check_status(self):
-        return self.process.poll() # Returns true if the process has terminated.
+        return self.process.poll() # Sets and returns the returncode attribute.
```

Comment:
The first explanation isn't exactly true. Based on the documentation poll() doesn't return a boolean value, it returns an integer which would be the returncode.